package.json

```diff
@@ -5,7 +5,7 @@
   "publishConfig": {
     "access": "public"
   },
-  "version": "1.6.0",
+  "version": "1.6.1-beta.1",
```

Comment:
not necessarily a patch, but I guess it doesn't matter how these beta versions are published?? will prob eventually be our ur 2.0

Reply:
ohh UR 2.0

Reply:
im just gonna send for now we can fix up later lol

Reply:
yan with 2.0 we can finally match up our npm versions with our UniversalRouter versions!